Frameworks/UIKit/StarboardXaml/ApplicationCompositor.cpp

```diff
@@ -38,8 +38,7 @@ void InitializeApp() {
     }
     initialized = true;
 
-    // Set our writable and temp folders
-    IwSetWritableFolder(Windows::Storage::ApplicationData::Current->LocalFolder->Path->Data());
+    // Set our temp folders
```

Comment:
with the single line code, the comment is not necessary anymore (in the past, this used to be several lines :))